incubating/service-now/step.yaml

```diff
@@ -0,0 +1,139 @@
+kind: step-type
+version: '1.0'
+metadata:
+  name: laurent-cf/service-now
```

Comment:
remove the account prefix

Reply:
I thought I had done that. I guess I forgot one.
It's why I like review